src/main/kotlin/racingcar/RacingApplication.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar
+
+fun main() {
+
+    RacingView.InputView.displayCarCount()
+    val carCount: String? = readLine()
+    require(!carCount.isNullOrBlank()) { IllegalArgumentException("잘못된 요청 입니다.") }
+
+    RacingView.InputView.displayRoundCount()
+    val roundCount: String? = readLine()
+    require(!roundCount.isNullOrBlank()) { IllegalArgumentException("잘못된 요청 입니다.") }
```

Comment:
```suggestion
    RacingView.InputView.displayCarCount()
    val carCount: String = readln()
    require(carCount.isNotBlank()) { "잘못된 요청 입니다." }

    RacingView.InputView.displayRoundCount()
    val roundCount: String = readln()
    require(roundCount.isNotBlank()) { "잘못된 요청 입니다." }
```

kotlin 1.6부터는 readLine() 대신 readln()을 사용할 수 있습니다. 요번 기회에 readLine()과 readln()의 차이를 찾아보세요.😊 

<img width="728" alt="스크린샷 2022-05-12 오후 9 27 49" src="https://user-images.githubusercontent.com/50702723/168074608-953cf86f-c916-4060-949d-5205e9644ddf.png">

require는 맞지 않으면, IllegalArgumentException을 보냅니다! 🤩 추가로 에러 메시지는 상수로 관리해 주세요!

참고 
- [kotlin readln](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/readln.html)
- [kotlin 1.6.0 released](https://blog.jetbrains.com/ko/kotlin/2021/12/kotlin-1-6-0-is-released/)

Reply:
자동차 요구사항 힌트에 있어서 사용해보았는데 readIn 구현체를 보니 제가 사용했던 것이 포함되어 있는 것 같네요~
?와 같은 연산자? 를 사용하지 않아도 되어서 좋네요 :)